device-type/overlay-car/frameworks/base/core/res/res/values/config.xml

```diff
@@ -13,5 +13,6 @@
     <!-- Disables the GnssTimeUpdate service. This is a switch for enabling Gnss time based
 	 suggestions to TimeDetector service. -->
     <bool name="config_enableGnssTimeUpdateService">false</bool>
+    <string name="config_defaultSms" translatable="true">com.android.car.messenger</string>
```

Comment:
Add comment what this config does. May be we can reuse the same comments from frameworks/base/core/res/res/values/config.xml.